scripts/ci/static_checks/pylint.sh

```diff
@@ -36,14 +36,21 @@ build_images::prepare_ci_build
 
 build_images::rebuild_ci_image_if_needed
 
+# Bug: Pylint only looks at PYLINTRC if it can't find a file in the _default_
```

Comment:
Not nice pylint :(

Reply:
Turns out it's even more annoying that that. PYLINTRC is an _additional_ pylint file.